lib/Provider/OAuth2.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace OCA\SocialLogin\Provider;
+
+use Hybridauth\User;
+
+class OAuth2 extends \Hybridauth\Adapter\OAuth2
+{
+    /**
+    * {@inheritdoc}
+    */
+    public function authenticate()
```

Comment:
Why you remove `authenticateCheckError` here ?

Reply:
I extended base `OpenID` because it does not handle `openid_mode` parameter in POST request. But PR with fix already merged to master https://github.com/hybridauth/hybridauth/pull/944
You no need to override it https://github.com/zorn-v/nextcloud-social-login/blob/master/3rdparty/hybridauth/hybridauth/src/Adapter/OAuth2.php#L290